_data/links.yml

```diff
@@ -2,6 +2,26 @@
   url: /join
   image: /assets/images/guild-colour-logo.png
 
-- text: Join our Discord!
+- text: Join our Discord
   url: /discord
   image: /assets/images/discord-logo.png
+
+- text: View upcoming events on our Instagram
+  url: /instagram
+  image: /assets/images/instagram-logo.png
+
+- text: "Find out about AFNOM's 'Missing Semester'"
+  url: https://missingsemester.afnom.net/
+  image: /assets/images/afnom_logo.png
+
+- text: Buy a CSS rubber duck
+  url: /ducks
+  image: https://www.guildofstudents.com/asset/Event/6531/CSSDuck.png
+
+- text: Meet our mascot: TeX
```

Comment:
Syntax error found here - should be wrapped in quotes (`- text: 'Meet our mascot: Tex'`, or alternatively `- text: Meet our mascot, Tex!`)

Reply:
Awesome! Thank u so much for doing the full check for this, I just found the build logs unreadable

Reply:
Just for convenience this was how I figured it out, it's not very clear but once you find it:

![image](https://github.com/CSSUoB/cssuob.github.io/assets/25277367/80887d29-fae1-496a-b4e2-8f639938f99e)